src/inc/msquic.h

```diff
@@ -812,6 +812,9 @@ void
 #define QUIC_PARAM_GLOBAL_EXECUTION_CONFIG              0x01000009  // QUIC_EXECUTION_CONFIG
 #endif
 #define QUIC_PARAM_GLOBAL_TLS_PROVIDER                  0x0100000A  // QUIC_TLS_PROVIDER
+#ifdef QUIC_API_ENABLE_PREVIEW_FEATURES
+#define QUIC_PARAM_GLOBAL_QUIC_OVER_TCP                 0x0100000B  // uint8_t
+#endif
```

Comment:
Instead, add a flag to `QUIC_EXECUTION_CONFIG_FLAGS`:
```c
typedef enum QUIC_EXECUTION_CONFIG_FLAGS {
    QUIC_EXECUTION_CONFIG_FLAG_NONE             = 0x0000,
#ifdef QUIC_API_ENABLE_PREVIEW_FEATURES
    QUIC_EXECUTION_CONFIG_FLAG_QTIP             = 0x0001,
#endif
} QUIC_EXECUTION_CONFIG_FLAGS;
```

Reply:
I think by doing this, we can eliminate all (most?) of the "TCP" code in the core logic. I'd really like to have zero TCP stuff in there, and this is just a pass through to the datapath.

Reply:
Can I just use MsQuicLib.ExecutionConfig.Flags everywhere in the code?

Reply:
Seems like all datapath code doesn't include library.h? So I still need to do some plumbing?